x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.ts

```diff
@@ -104,6 +105,20 @@ export const isolationRequestHandler = function (
     }
     agentIDs = [...new Set(agentIDs)]; // dedupe
 
+    // convert any alert IDs into cases
+    let caseIDs: string[] = req.body.case_ids?.slice() || [];
+    if (req.body.alert_ids && req.body.alert_ids.length > 0) {
+      const newIDs: string[][] = await Promise.all(
+        req.body.alert_ids.map(async (a: string) =>
+          (await endpointContext.service.getCasesClient(req, context)).getCaseIdsByAlertId({
```

Comment:
Saw this after the fact but do we need the `await` inside of the `Promise.all`?

Reply:
I think we do. Switching to `getCasesClientWithRequestAndContext` required this `getCasesClient` to need to be an async function, and we need to call `.getCaseIdsByAlertId()` on the awaited object.

There's probably a more elegant way to deal with this, but this is throwaway stuff right now anyway

Reply:
(also without the await, I got `cannot call getCaseIdsByAlertId on undefined` so I def needed it, there just may have been a more elegant way to _do_ that)